server/app/controllers/dev/ProfileController.java

```diff
@@ -27,6 +27,6 @@ public Result index(Http.Request request) {
         maybeProfile
             .map((p) -> JsonPrettifier.asPrettyJsonString(p.getProfileData()))
             .orElse("No profile present");
-    return ok(profileContent).as(Http.MimeTypes.TEXT);
+    return ok(String.format(profileContent)).as(Http.MimeTypes.JSON);
```

Comment:
```suggestion
    return ok(profileContent).as(Http.MimeTypes.JSON);
```

This isn't doing anything now so let's take it out.